rx-receivers/src/main/java/com/f2prateek/rx/receivers/battery/RxBatteryManager.java

```diff
@@ -0,0 +1,42 @@
+package com.f2prateek.rx.receivers.battery;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.BatteryManager;
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import com.f2prateek.rx.receivers.RxBroadcastReceiver;
+import rx.functions.Func1;
+import rx.Observable;
+import static com.f2prateek.rx.receivers.internal.Preconditions.checkNotNull;
+
+public class RxBatteryManager {
+  private RxBatteryManager() {
+    throw new AssertionError("no instances");
+  }
+
+  /** TODO: docs. */
+  @CheckResult @NonNull //
+  public static Observable<Integer> status(@NonNull final Context context) {
```

Comment:
Actually, reading the docs looks like this Intent contains a lot more information than just `EXTRA_STATUS` — https://cloudup.com/cIexa7GOiHZ.

It should be wrapped into a class like here https://github.com/f2prateek/rx-receivers/blob/master/rx-receivers/src/main/java/com/f2prateek/rx/receivers/wifi/SupplicantStateChangedEvent.java